src/main/java/duke/command/AddCommand.java

```diff
@@ -42,4 +42,16 @@ public void execute(TaskList taskList, Ui ui, Storage storage) {
     public boolean isExit() {
         return false;
     }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
Nice equals method syntax 👌

Reply:
Thanks for the suggestion! Following this format would make the code neater and more standardised.